pkg/pluginmanager/manager.go

```diff
@@ -759,14 +759,17 @@ func SyncPlugins() error {
 		// Note: to install all plugins for a specific product, plugin groups will
 		// need to be used.
 		plugins, err = DiscoverServerPlugins()
+		if err != nil {
+			return err
+		}
```

Comment:
The error was mistakenly being ignored before.
This should be a separate PR, but in the interest of time, I'm cheating a bit.

Reply:
Thanks for fixing this. 👍

Reply:
Thanks for fixing this. 👍